### ISSUE TITLE
Example Data

### ISSUE THREAD

**Issue by spficklin** [SystemsGenetics/GEMmaker#19]
To complete our example data we need the following
1)  A set of paired files
2)  A remote dataset that is not large but which generates FPKM values.
3)  For our SciDAS project a larger set of 5-10 Arabidopsis datasets that represent the variety of Illumina datasets we might encounter (small, large, different illumina platforms, high contamination.)

**Comment by spficklin** [comment]
Also, it may be a good idea to update the nextflow.config.example file to have the settings for both local and remote example types but with the remote entries commented out.  This way user's just have to comment one out and uncomment the other... It just makes it a bit easier to test.

**Comment by spficklin** [comment]
I'm assigning this to @JohnHadish as he was working on the unit test dataset. @JohnHadish can you update this issue as to the status?

**Comment by JohnHadish** [comment]
Changed on SALMON_AND_KALLISTO branch. The remote file and the local files now use the same reference "genome". I added 3 extra genes which map to the remote file to the "genome" so that values will be generated. All are very small files that run quickly.
The unit_test consists of 7 runs, consisting of paired, unpaired, divided sample and whole sample. They are typical sized files that you find on NCBI.
